deploy/load-generator.yaml

```diff
@@ -21,7 +21,7 @@ spec:
         imagePullPolicy: IfNotPresent
         env:
         - name: ENDPOINT
-          value: "http://book-svc:8080/"
+          value: "http://app.example.com/"
```

Comment:
Shouldn't this remain book-svc only ?

Reply:
This is not working for header based routing. We need to access application through ingress endpoint.